core/src/main/java/org/elasticsearch/action/admin/cluster/node/tasks/list/TaskInfo.java

```diff
@@ -50,17 +51,24 @@
 
     private final String description;
 
+    private final long startTime;
```

Comment:
This is just a transport format, which is easier to work with. We do convert this long to a TimeValue in [XContentBuilder#timeValueField()](https://github.com/elastic/elasticsearch/pull/16829/files#diff-2fdeeb0601d461c70295ca0a00059276R967) if a user requests `human` readable output. It also happens to dates in [XContentBuilder#dateValueField()](https://github.com/elastic/elasticsearch/pull/16829/files#diff-2fdeeb0601d461c70295ca0a00059276R975). So, I don't think storing it here in milliseconds and nanoseconds will affect readability.